### DISCUSSION TITLE
admin.shopify.com compatibility

### DISCUSSION THREAD

**Discussion by ZainSohail** [Kyon147/laravel-shopify#32] [Q&A]
Really love the project and glad to see its progress is still being maintained.

I just received a response from Shopify that app is not working on admin.shopify.com domain, I'm using blade templates. I just did my quick search and found [#1250](https://github.com/gnikyt/laravel-shopify/issues/1250) on the old repo, I'm not sure whether this has been fixed in this repo or not, can someone who has faced this issue guide me further on this? 

Its frustrating how Shopify would throw us into this when we cannot even test our apps in this domain, test stores are still using the old domain structure.

Any help is appreciated, TIA.

**Comment by Kyon147** [comment]
`17.4.2` should be fine for you now as that deals with the `host` updates.

**Reply by asada-chetu** [reply]
Hi @Kyon147

I am creating the Shopify app using the ossiet package 17.4.2 but we are getting the blank page issue when switching the navigation by <a> tag.

Laravel: 8.x
Ossiet: 17.4.2
Frontend: Blade

**Reply by qasim-glowlogix** [reply]
@asada-chetu were you able to fix this issue? I am facing the same issue tried everything i could.

**Reply by Kyon147** [reply]
Hi, can you try the latest version as their were some more host updates in the newest one.
